dev/io.openliberty.org.jboss.resteasy.common/bnd.bnd

```diff
@@ -128,6 +128,7 @@ Import-Package: \
   !com.sun.xml.bind.marshaller, \
   !org.apache.cxf.*, \
   !org.apache.james.mime4j.*, \
+  !org.eclipse.microprofile.config.*, \
```

Comment:
Rather than being removed completely, doesn't this become an optional dependency on MP Config 3.0?

Reply:
This line will prevent the resteasy.common bundle from importing MP Config packages.

Reply:
if someone does enable the mpConfig-3.0 feature, shouldn't resteasy be able to use it as well? We can address that as a follow-up though